ReactAndroid/src/main/java/com/facebook/react/common/BUCK

```diff
@@ -33,6 +33,7 @@ rn_android_build_config(
     package = "com.facebook.react",
     values = [
         "boolean IS_INTERNAL_BUILD = true",
+        "int DEBUG_SERVER_HOST_PORT = 8081",
```

Comment:
I had to do this to get unit tests to pass. Not sure if this is the best approach.